Core/src/org/sleuthkit/autopsy/discovery/ui/DomainDetailsPanel.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Autopsy
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.discovery.ui;
+
+import com.google.common.eventbus.Subscribe;
+import java.awt.Component;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import org.apache.commons.lang.StringUtils;
+import org.sleuthkit.autopsy.discovery.search.DiscoveryEventUtils;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.autopsy.discovery.search.SearchData;
+
+/**
+ * Panel to display details area for domain discovery results.
+ *
+ */
+final class DomainDetailsPanel extends JPanel {
+
+    private static final long serialVersionUID = 1L;
+    private static ArtifactsWorker detailsWorker;
+    private static String domain;
+    private static String selectedTabName;
+
```

Comment:
Domain should definitely be an instance variable and was a mistake where I didn't stop to think about what it should be.
ArtifactsWorker I intended to ensure it was cancelled if the user decided to perform another search although it should be destroyed when this panel is destroyed so that should be unnecessary so I will switch it to a instance variable.
selectedTabName was intended to be an attempt to preserve the most recently selected tab between searches but it apparently isn't working so I am going to try storing it else where and see if that works as intended either way it will become an instance variable as well.